Replace WooCommerce logo with Woo 2.0 Brand

eason9487 opened the pull request:
### Changes proposed in this Pull Request:

PT: peeuvX-1Wl-p2

This PR replaces WooCommerce logo to Woo 2.0 Brand.


### 📌 Checklist before merging (@eason9487)

- [x] Confirm the official launch date in PT peeuvX-1Wl-p2 or paqDfi-bqN-p2
   - The official launch date is February 4, 2025
- [x] The changes has been reviewed and approved by the product team
   - They will process this on a staging site and update the statutes on their worksheets
- [x] Wait until the official launch date is near
   - Can be merged between January 30 and February 3

### Screenshots:

#### 📷 Get Started page

Before v.s. After

![1](https://github.com/user-attachments/assets/0aaae8d3-4b39-4ed8-abc0-11878db995d7)

#### 📷 Submission success modal on Product Feed page

Before v.s. After

![2](https://github.com/user-attachments/assets/f9f71e82-ab31-4fd5-a2ab-1ec6293bc7e5)

### Detailed test instructions:

1. Go to Get Started page and view the Woo logo in the third card
2. Go to Product Feed page with path `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fproduct-feed&guide=submission-success` and view the WooCommerce logo in Submission success modal

### Changelog entry

> Tweak - Replace WooCommerce logo with Woo 2.0 Brand.

eason9487 committed:
Update the Woo logo in the img-free-listings.svg file to Woo 2.0 Brand.

eason9487 committed:
Update the woocommerce-logo.svg file to Woo 2.0 Brand.

ianlin reviewed (approved):
Thanks for changing the Woo logo. Confirmed the new logo were displayed correctly, and I didn't see other places have Woo logo that required to change.

eason9487 committed:
Adjust a WooCommerce logo according to feedback from the product team.

Feedback:
- Increase WooCommerce logo size by 25%.
- Decrease Google logo size by 20%.
- Align the baseline of logos.
- Center the entire grouping (including Google logo and the separator)
  so that the space on the left and right of the grouping is the same.
- The separator won't be in the middle after the adjustment.
